### DISCUSSION TITLE
<Your question or feedback about the GitHub Discussions product>

### DISCUSSION THREAD

**Discussion by Asabi89** [community/community#133324] [Programming Help]
### Select Topic Area

Question

### Body

_a good Samaritan there come and help me I have a Django code that I want reorganize_

**Comment by LiteBrite82** [comment]
Thanks for posting in the GitHub Community, @Asabi89 ! 

We’ve moved your post to our Programming Help 🧑‍💻 category, which is more appropriate for this type of discussion.

Please review our [guidelines](https://github.com/orgs/community/discussions/51384) about the Programming Help category for more information.
